core/domain/exp_jobs_one_off.py

```diff
@@ -535,3 +536,41 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class ImageDataMigrationJob(jobs.BaseMapReduceOneOffJobManager):
+    """ One-off job for migrating the images in the exploration
+        from the GAE to GCS
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [file_models.FileSnapshotContentModel]
+
+    @staticmethod
+    def map(file_snapshot_content_model):
+        instance_id = file_snapshot_content_model.get_unversioned_instance_id()
+        filetype = instance_id[instance_id.rfind(".") + 1:]
```

Comment:
Use single quotes in backend, throughout.

Reply:
Done!